app/locales/en-us/translations.js

```diff
@@ -280,7 +277,7 @@ const translations = {
                         "label": "I react annoyed if another person steals the show from me."
                     }
                 },
-                "label": "15. Please rate the extent to which you agree or disagree with the following statements:",
+                "label": "16. Please rate the extent to which you agree or disagree with the following statements:",
```

Comment:
The `{{index}}` starts at zero so I would need to write a helper to get index + 1 -- does that sound like an okay approach @samchrisinger?